lib/tags.js

```diff
@@ -120,7 +120,8 @@ function getTags ({
     { tag: 'amp-web-push', version: '0.1' },
     { tag: 'amp-wistia-player', version: '0.1' },
     { tag: 'amp-yotpo', version: '0.1' },
-    { tag: 'amp-youtube ', version: '0.1' }
+    { tag: 'amp-youtube ', version: '0.1' },
+    { tag: 'amp-story ', version: '1.0' }
```

Comment:
`amp-story` already exists on line [104](https://github.com/nuxt-community/amp-module/blob/842c01f31249c7db37c4c0942e967ec1c7558352/lib/tags.js#L104)

Reply:
This is my mistake it already removes